src/fragments/lib/troubleshooting/common/upgrading.mdx

```diff
@@ -842,7 +842,250 @@ For a deeper look at how the Storage functionality in V6, check out our [Storage
+
+Expand the sections below to see the events that have been changed or removed. There have been no changes to API, DataStore, and PubSub events.
+
+<Accordion title="Auth Hub Events" headingLevel="4">
+  <table>
+    <thead>
+      <tr>
+        <th colSpan="2">V5</th>
+        <th colSpan="2">V6</th>
+      </tr>
+      <tr>
+        <th>Event</th>
+        <th>Data</th>
+        <th>Event</th>
+        <th>Data</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>configured</td>
+        <td>null</td>
+        <td>Use the new 'configure' event from the 'core' channel</td>
+        <td>ResourcesConfig</td>
+      </tr>
+      <tr>
+        <td>signIn</td>
+        <td>CognitoUser</td>
+        <td>signedIn</td>
+        <td>AuthUser</td>
+      </tr>
+      <tr>
+        <td>signIn_failure</td>
+        <td>error</td>
+        <td></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>signUp</td>
+        <td>ISignUpResult</td>
+        <td></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>signUp_failure</td>
+        <td>error</td>
+        <td></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>confirmSignUp</td>
+        <td>'SUCCESS'</td>
+        <td></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>completeNewPassword_failure</td>
+        <td>error</td>
+        <td></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>autoSignIn</td>
+        <td>CognitoUser</td>
+        <td></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>autoSignIn_failure</td>
+        <td>error</td>
+        <td></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>forgotPassword</td>
+        <td>CognitoUser</td>
+        <td></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>forgotPassword_failure</td>
+        <td>error</td>
+        <td></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>forgotPasswordSubmit</td>
+        <td>CognitoUser</td>
+        <td></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>forgotPasswordSubmit_failure</td>
+        <td>error</td>
+        <td></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>verify</td>
+        <td>CognitoUser</td>
+        <td></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>tokenRefresh</td>
+        <td></td>
+        <td>tokenRefresh</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>tokenRefresh_failure</td>
+        <td>error</td>
+        <td>tokenRefresh_failure</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>cognitoHostedUI</td>
+        <td>CognitoUser</td>
+        <td></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>cognitoHostedUI_failure</td>
+        <td>error</td>
+        <td></td>
```

Comment:
here should be the `signInWithRedirect_failure` event, which includes `error` in the payload